app/Repositories/Booking/BookingRepositoryInterface.php

```diff
@@ -5,4 +5,8 @@
 interface BookingRepositoryInterface
 {
     public function updateStatus($id, $status);
+    public function updateDeposit($id, $deposit);
+    public function countByStatus($status);
+    public function getWithOrderBy($attributes = [], $column, $key);
+    public function checkRoomAvailable($room_id, $checkin, $checkout);
```

Comment:
comment vào các function 
```suggestion
/**
     * mục đích
     * @param  
     * @return 
     */
    public function updateStatus($id, $status);
```